# A single parameter file for all nodes in the launch file.
# The top-level key must match the node name defined in the launch file.

tca_node_one:
  ros__parameters:
    i2c_bus: "/dev/i2c-1"
    i2c_address: 112 # 0x70

tca_node_two:
  ros__parameters:
    i2c_bus: "/dev/i2c-1"
    i2c_address: 113 # 0x71

tca9548a_manager:
  ros__parameters:
    # A list of I2C addresses (in decimal) for the devices to manage.
    # The manager will use these to create service clients.
    i2c_addresses: [112, 113]